config/config_test.go

```diff
@@ -534,12 +694,8 @@ flannel:
 
 	for i, test := range tests {
 		cfg, _, err := Parse([]byte(test.in.data))
-		if !reflect.DeepEqual(err, test.out.r) {
-			t.Errorf("#%d: bad error: want %v, got %v", i, test.out.r, err)
-		}
-		if !reflect.DeepEqual(cfg, test.out.cfg) {
-			t.Errorf("#%d: bad config: want %#v, got %#v", i, test.out.cfg, cfg)
-		}
+		assert.Equal(t, test.out.r, err, "#%d: bad report", i)
```

Comment:
Does testify print the want/got messages?

Reply:
Yes, and it also pretty-prints a nice diff between the expected and actual structs, which was the motivation here for me to add it. Comparing the difference between walls of text is slow and hard when attempting to figure out issues.

Reply:
Turns out the packages I `ignore`'d earlier were necessary for `testify`, so I just readded them in a new commit for the `config/config_test.go` changes.